cognite/client/_api/workflows.py

```diff
@@ -212,6 +212,7 @@ def list(
         workflow_version_ids: WorkflowVersionIdentifier | MutableSequence[WorkflowVersionIdentifier] | None = None,
         created_time_start: int | None = None,
         created_time_end: int | None = None,
+        statuses: list[Literal["completed", "failed", "running", "terminated", "timed_out"]] | None = None,
```

Comment:
Can you also allow a single status to be passed in, not just a list? It is a minor thing, but makes the users life a bit easier

Reply:
Any reason why this list is different than? https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/data_classes/workflows.py#L24

Reply:
The statuses there are the statuses for Tasks. The naming is a bit unfortunate. Would be a potential breaking change.